internal/fixture/fixture.go

```diff
@@ -359,7 +359,6 @@ func FixBindingWithInstanceID(bindingID string, instanceID string) internal.Bind
 
 		Kubeconfig:        "kubeconfig",
 		ExpirationSeconds: 600,
-		GenerationMethod:  "adminkubeconfig",
```

Comment:
If the `GenerationMethod` was excessive and is now removed, we should use values `adminkubeconfig` or `tokenrequest` for BindingType field (to be consistent with description of the database field).